mithril-relay/src/p2p/peer.rs

```diff
@@ -75,28 +89,36 @@ impl Peer {
     }
 
     fn build_topics() -> HashMap<TopicName, gossipsub::IdentTopic> {
-        HashMap::from([(
-            MITHRIL_SIGNATURES_TOPIC_NAME.into(),
-            gossipsub::IdentTopic::new(MITHRIL_SIGNATURES_TOPIC_NAME),
-        )])
+        HashMap::from([
+            (
+                MITHRIL_SIGNATURES_TOPIC_NAME.into(),
```

Comment:
Maybe the topic could be gathered in a mod (something like that):
```rust
pub mod mithril_p2p_topic {
  pub const SIGNATURES: &str = "mithril/signatures";
  pub const SIGNERS: &str = "mithril/signers";
}
```

Reply:
Done